docs/workflows.md

```diff
@@ -9,3 +9,22 @@ If you need to add a new package to the Neon project please make sure to execute
 4. Add a Symlink to our main [license](../assets/AGPL-3.0.txt).
 5. Update [commitlint.yaml](../commitlint.yaml) by adding the new package name.
 6. Remove the package `.gitignore` (a global `.gitignore` for all packages is used instead).
+
+## Publishing new versions of the packages
+
+Because [melos does not support signing-off](https://github.com/invertase/melos/issues/591) yet the process is a little more complicate:
```

Comment:
```suggestion
Because [melos does not support signing-off](https://github.com/invertase/melos/issues/591) yet the process is a little more complicated:
```